packages/docs/docs/tutorials/browse-sample-data.md

```diff
@@ -0,0 +1,32 @@
+
+| Data Type                                                                                                                                                                                                                                                                                                                                                                                   | Browse on Medplum                                                                                                                                                                                                                                                                                                                                       | Related Documentation                                                                                                                                                                        |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Patient Demographics, including: <br /> ● Birth Sex <br /> ● Date of Birth <br /> ● Race <br /> ● Ethnicity <br /> ● Preferred Language <br /> ● Address                                                                                                                                                                                                                                    | [Patient](https://app.medplum.com/Patient?_count=20&_fields=_lastUpdated,name,birthDate,gender,telecom,address&_offset=0&_sort=-_lastUpdated)                                                                                                                                                                                                           | [Medplum App](/docs/app)                                                                                                                                                                     |
```

Comment:
I think this table is great. My one concern is that it may come across as a bit too big and intimidating for an intro tutorial. @reshmakh how would you feel about removing the sub-bullets in the first column for the tutorial version of this table? Since the user is going to browse the data anyway, I don't think calling out individual fields adds much.

Reply:
Good idea, updating.